database/variables.tf

```diff
@@ -33,7 +33,14 @@ variable "tags" {
 
 variable "json_params" {
   description = "A JSON string of arbitrary parameters"
-  type        = map(object)
-  default     = null
+  type = map(object({
```

Comment:
Will this require specifying all of these elements? Or will they remain optional?